### ISSUE TITLE
Fix OS X 10.11.4 breaking changes

### ISSUE THREAD

**Issue by ptmt** [ptmt/react-native-desktop#31]
`NSMenuItem` has no longer supports `submenu` property?

**Comment by beretboat** [comment]
Just tried, `submenu` worked. I'm still on 10.11.3, but I don't think it's the same on 10.11.4. You might need to call `addItem:` besides `setSubmenu`:
```objc
[rootMenu setSubmenu:submenu forItem:item];
[rootMenu addItem:item];
```

**Comment by ptmt** [comment]
Yeah, it's only for 10.11.4. As soon Travis will support the latest `osx_image`, I add it to the tests configuration.

**Comment by ptmt** [comment]
It was like AppKit bug maybe? In released 10.11.4 no such thing, everything works.
